exercises/queen-attack/src/test/java/QueenAttackCalculatorTest.java

```diff
@@ -0,0 +1,114 @@
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertTrue;
+
+public final class QueenAttackCalculatorTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
```

Comment:
I like the second option - we can save a search and provide a link to: https://github.com/junit-team/junit4/wiki/Rules, but still leave reading and discovery to the user?